src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs

```diff
@@ -96,7 +89,7 @@ public void UpdateWithWorkloadSets()
 
             newRollback.ManifestVersions.Should().NotBeEquivalentTo(rollbackAfterUpdate.ManifestVersions);
 
-            GetWorkloadVersion().Should().Be("8.0.201");
+            GetWorkloadVersion().Should().Be("8.0.300-preview.0.24217.2");
```

Comment:
No, you have to set up the VM state to include the workload set packages in the folder.

The later test is supposed to fail because the command uses `--source` to point to the local folder, which means it won't find the manifests that are available on NuGet.org.